frontend/src/metabase/admin/upsells/UpsellPermissions.tsx

```diff
@@ -0,0 +1,31 @@
+import { t } from "ttag";
+
+import { getPlan } from "metabase/common/utils/plan";
+import { useSelector } from "metabase/lib/redux";
+import { getSetting } from "metabase/selectors/settings";
+
+import { UpsellBanner } from "./components";
+
+export const UpsellPermissions = ({ source }: { source: string }) => {
+  const plan = useSelector(state =>
+    getPlan(getSetting(state, "token-features")),
+  );
+
+  const showUpsell = plan === "oss" || plan === "starter";
+
+  if (!showUpsell) {
+    return null;
+  }
```

Comment:
some of the other upsells look like they check if the user has the feature enabled or not. should we check if the user has the advanced permissions enabled before showing the upsell?

Reply:
Makes sense. Thanks
https://github.com/metabase/metabase/pull/53080/commits/89f50bd5e57348dcaa6de791672fb06327d0e9ad